.env

```diff
@@ -1 +1,2 @@
 VITE_MEDIASTORE_SDK_VERSION=$npm_package_version
+VITE_RECAPTCHA_SITE_KEY="6Ld0A54qAAAAANJ8mLCpJAxEp0XKtJyueFmEFVaG"
```

Comment:
@pawelacio, this will make captcha work on all environments. do we want that? or do we want to only enable captcha on production?

if we want to enable it only on prod, please remove the entry from this file and add it to all the `.env.{environment}` files separately. so in `.env.staging` i suggest adding `VITE_RECAPTCHA_SITE_KEY=""` - an empty string. the same for `.env.sandbox`. then, in `.env.production`, you could provide the real value to make it work only in prod

Reply:
I think that we can merge this one like this (we are merging this to reCAPTCHA branch not to block the stage branch) and I will add that comment into one of other ticket (for example for the purchase flow). I think that maybe for the **staging** and **sandbox** we will add some testing kay that will always be valid but its something worth checking (i will try to do it after the Christmas break)